opentelemetry/ext/honeycomb/__init__.py

```diff
@@ -81,6 +81,31 @@ def shutdown(self):
         self.client = None
 
 
+class HoneycombConsoleSpanExporter(SpanExporter):
+    """
+    Implementation of :class:`SpanExporter` that prints spans to the console in a
+    format edible by the Honeycomb AWS Lambda Instrumentation.
+    """
+
+    def __init__(
+        self,
+        service_name: typing.Optional[str] = None,
```

Comment:
Would you mind remove the typing in this section? I don't think we use it elsewhere in the repo, and we'd prefer to following the existing code style to help maintain consistency.

Reply:
Done!